### ISSUE TITLE
Consider using bazelisk to automatically download and use the correct Bazel version

### ISSUE THREAD

**Issue by vkuzmin-uber** [uber/neuropod#247]
TL;DR: Use Bazelisk https://github.com/bazelbuild/bazelisk and .bazelversion to build neuropod.

Neuropod requires specific Bazel version:

build/install_system_deps.sh
....
curl -sSL -o bazel.sh https://github.com/bazelbuild/bazel/releases/download/0.28.1/bazel-0.28.1-installer-darwin-x86_64.sh

I found reference to *bazelisk* in Release Notes of TF https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0-rc1:

"If you're building Tensorflow from source, consider using bazelisk to automatically download and use the correct Bazel version. Bazelisk reads the .bazelversion file at the root of the project directory."

It can use either .bazelversion file or env variable, something like:

USE_BAZEL_VERSION=$BAZEL_VERSION BAZEL_USE_CPP_ONLY_TOOLCHAIN=1 bazelisk --output_base $bazel_cache build $BAZEL_FLAGS //neuropods:all //neuropods:packages
